docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/ConvertTypeMultiColumn.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    // This example illustrates how to convert multiple columns of different types to one type, in this case System.Single. 
+    // This is often a useful data transformation before concatenting the features together and passing them to a particular estimator.
+    public static class ConvertTypeMultiColumn
+    {
+        // The initial data type
+        private class InputData
+        {
+            public bool Feature1;
+            public string Feature2;
+            public DateTime Feature3;
+            public double Feature4;
+        }
+
+        // The resulting data type after the transformation
+        private sealed class TransformedData : InputData
+        {
+            public float Converted1 { get; set; }
+            public float Converted2 { get; set; }
+
+            public float Converted3 { get; set; }
+            public float Converted4 { get; set; }
+        }
+
+        public static void Example()
+        {
+            var mlContext = new MLContext(seed: 1);
```

Comment:
>var mlContext = new MLContext(seed: 1); [](start = 12, length = 39)

I see missing comments overall in this sample which we usually have in other samples e.g. comments above the MLContext etc.